tests/lib/config/flat-config-array.js

```diff
@@ -2264,4 +2264,27 @@ describe("FlatConfigArray", () => {
         });
 
     });
+
+    describe("invalid options", () => {
+
+        it("should report key when an invalid option is found during merge", () => {
```

Comment:
Isn't this the same as the above `should error when an unexpected value is found` test that changed?

Reply:
Oops, yes, thought I had removed this. Thanks.